src/config.ml

```diff
@@ -36,6 +36,29 @@ open Stanza.Decoder
    files for simplicity *)
 let syntax = Stanza.syntax
 
+module Build_logs = struct
```

Comment:
Is `Build_logs` a good name?  I couldn't think of anything better, but I'm not super happy with it

Reply:
Agreed. In my head "build logs" means the logs file, i.e. `_build/log`. I have a feeling that it might be the same for other users as well. What about one of these:
- `on-rebuild=ACTION`
- `terminal-clearing-mode=MODE`
?